src/operator/nn/fully_connected.cc

```diff
@@ -214,9 +238,7 @@ If ``no_bias`` is set to be true, then the ``bias`` term is ignored.
 .set_attr<nnvm::FInferShape>("FInferShape", FullyConnectedShape)
 .set_attr<nnvm::FInferType>("FInferType", FullyConnectedType)
 .set_attr<FCompute>("FCompute<cpu>", FullyConnectedCompute<cpu>)
-#if MXNET_USE_MKLDNN == 1
```

Comment:
Why remove this?

Reply:
This is for adding sparse support for FC. See line 213.

Reply:
I see. Both sparse and mkldnn support need run into this dispatch.